### PR TITLE
Conditional menu build fix for window

### PR DESCRIPTION
**Pull request by ppot**

### CODE REVIEW

**Comment by ppot** [comment]
@Stanzilla You can test it.

**Comment by Stanzilla** [comment]
Fixes it!

File: app/menus/menu.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ const darwinMenu = require('./menus/darwin');
 module.exports = (createWindow, updatePlugins) => {
   const commands = getKeymaps().commands;
   const menu = [
-    (process.platform === 'darwin' ? darwinMenu(commands) : []),
```
Comment (albinekb) [inline app/menus/menu.js]: ...(process.platform === 'darwin' ? [darwinMenu(commands)] : []),

Comment (albinekb) [inline app/menus/menu.js]: you copy pasted it wrong 😉 https://github.com/zeit/hyper/pull/1876#discussion_r118828269

Comment (albinekb) [inline app/menus/menu.js]: oh my comment was wrong too :D 
`...(process.platform === 'darwin' ? [darwinMenu(commands)] : []),` this should work

Comment (Stanzilla) [inline app/menus/menu.js]: I tried `(process.platform === 'darwin' ? [darwinMenu(commands)] : []),` and it still causes a blank menu entry on Windows

Comment (chabou) [inline app/menus/menu.js]: you should keep '...' in the beginning to destructure (possible empty) array.

Comment (Stanzilla) [inline app/menus/menu.js]: oh! that makes it work indeed

Comment (ppot) [inline app/menus/menu.js]: ok, will update

**Comment by albinekb** [comment]
Nice @chabou  @ppot 💯 '

**Pull request merged**